[2023-10-17T11:19:07.904+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-04T00:00:00+00:00 [queued]>
[2023-10-17T11:19:07.913+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-04T00:00:00+00:00 [queued]>
[2023-10-17T11:19:07.913+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:07.951+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-04 00:00:00+00:00
[2023-10-17T11:19:07.958+0000] {standard_task_runner.py:57} INFO - Started process 551 to run task
[2023-10-17T11:19:07.960+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-04T00:00:00+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpaeiwdvn6']
[2023-10-17T11:19:07.962+0000] {standard_task_runner.py:85} INFO - Job 67: Subtask waiting_for_data
[2023-10-17T11:19:08.006+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-04T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:08.076+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-04T00:00:00+00:00'
[2023-10-17T11:19:08.083+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:08.083+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:08.084+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:08.084+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:08.090+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210304T000000, start_date=20231017T111907, end_date=20231017T111908
[2023-10-17T11:19:08.132+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:08.145+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:35.638+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-04T00:00:00+00:00 [queued]>
[2023-10-18T10:42:35.647+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-04T00:00:00+00:00 [queued]>
[2023-10-18T10:42:35.648+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:35.706+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-04 00:00:00+00:00
[2023-10-18T10:42:35.713+0000] {standard_task_runner.py:57} INFO - Started process 1514 to run task
[2023-10-18T10:42:35.716+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-04T00:00:00+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp0gqv6xfd']
[2023-10-18T10:42:35.717+0000] {standard_task_runner.py:85} INFO - Job 66: Subtask waiting_for_data
[2023-10-18T10:42:35.759+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-04T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:35.826+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-04T00:00:00+00:00'
[2023-10-18T10:42:35.840+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:35.843+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210304T000000, start_date=20231018T104235, end_date=20231018T104235
[2023-10-18T10:42:35.852+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 66 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1514)
[2023-10-18T10:42:35.892+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:35.915+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:05.447+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-04T00:00:00+00:00 [queued]>
[2023-10-18T11:17:05.458+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-04T00:00:00+00:00 [queued]>
[2023-10-18T11:17:05.458+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:05.473+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-04 00:00:00+00:00
[2023-10-18T11:17:05.481+0000] {standard_task_runner.py:57} INFO - Started process 295 to run task
[2023-10-18T11:17:05.486+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-04T00:00:00+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp4t2zpfa0']
[2023-10-18T11:17:05.487+0000] {standard_task_runner.py:85} INFO - Job 86: Subtask waiting_for_data
[2023-10-18T11:17:05.554+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-04T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:05.635+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-04T00:00:00+00:00'
[2023-10-18T11:17:05.644+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:05.645+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:05.645+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:05.645+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:05.656+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210304T000000, start_date=20231018T111705, end_date=20231018T111705
[2023-10-18T11:17:05.696+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:05.740+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:11:44.562+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-04T00:00:00+00:00 [queued]>
[2023-10-19T12:11:44.572+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-04T00:00:00+00:00 [queued]>
[2023-10-19T12:11:44.573+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:11:44.589+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-04 00:00:00+00:00
[2023-10-19T12:11:44.596+0000] {standard_task_runner.py:57} INFO - Started process 356 to run task
[2023-10-19T12:11:44.599+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-04T00:00:00+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpslwrw0zi']
[2023-10-19T12:11:44.600+0000] {standard_task_runner.py:85} INFO - Job 85: Subtask waiting_for_data
[2023-10-19T12:11:44.645+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-04T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:11:44.729+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-04T00:00:00+00:00'
[2023-10-19T12:11:44.741+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:11:44.742+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:11:44.742+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:11:44.743+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:11:44.755+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210304T000000, start_date=20231019T121144, end_date=20231019T121144
[2023-10-19T12:11:44.813+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:11:44.850+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
